tutorials/layout/index.md

```diff
@@ -0,0 +1,1003 @@
+[Elevation and
+Shadows](https://material.io/guidelines/material-design/elevation-shadows.html)
+in the [material guidelines](https://material.io/guidelines/).
+Specifying an unsupported value disables the drop shadow entirely.
+
+#### Card summary:
+
+* Implements a [material design
+  card](https://material.io/guidelines/components/cards.html)
+* Used for presenting related nuggets of information
+* Accepts a single child, but that child can be a Row, Column, or other
+  widget that holds a list of children
+* Displayed with rounded corners and a drop shadow
+* A Card's content can't scroll
+* From the material library
+
+#### Card examples:
+
+<div class="row"> <div class="col-md-6" markdown="1">
+
+<img src="images/card.png" style="border:1px solid black" alt="xx">
+
+Card containing 3 ListItems. A Divider separates the first and second ListItem.
+The icons are from the list of supported
+[Icons](https://docs.flutter.io/flutter/material/Icons-class.html).
+
+**Code:** below
+
+</div> <div class="col-md-6" markdown="1">
+
+<img src="images/card-flutter-gallery.png" style="border:1px solid black" alt="xx">
+
+A Card containing an image and text.<br>
+**Code:** [cards_demo.dart](https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/cards_demo.dart)
+from the [Flutter
+Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery)
+
+</div> </div>
+
+{% include includelines filename="_code/card/main.dart" start=40 count=33 %}
+
+<hr>
+
+### ListItem
+
+Use
+ListItem, a specialized row widget from the material library, for an easy
+way to create a row containing up to 3 lines of text and optional leading
+and trailing icons. ListItem is most commonly used in Card or ListView,
+but can be used elsewhere.
+
+#### ListItem summary:
+
+* A specialized row that contains up to 3 lines of text and optional icons
+* Less configurable than Row, but easier to use
+* From the material library
+
+#### ListItem examples:
+
+<div class="row"> <div class="col-md-6" markdown="1">
+
+<img src="images/card.png" style="border:1px solid black" alt="xx">
+
+A Card containing 3 ListItems.<br>
+**Code:** See [Card examples](#card-examples).
+
+</div> <div class="col-md-6" markdown="1">
+
+<img src="images/listitem-flutter-gallery.png" style="border:1px solid black" alt="xx">
+
+Uses ListItem to list 3 drop down button types.<br>
+**Code:** [buttons_demo.dart](https://github.com/flutter/flutter/blob/master/examples/flutter_gallery/lib/demo/buttons_demo.dart)
+from the [Flutter
+Gallery](https://github.com/flutter/flutter/tree/master/examples/flutter_gallery)
+
+</div> </div>
+
+<hr>
+
+<a name="building"></a>
+## Building a layout
+
+The previous sections describe Flutter's approach to creating a layout.
+How do you use this knowledge to build an entire layout?
+Let's walk through the layout for the following screenshot:
+
+<img src="images/lakes.png" style="border:1px solid black" alt="">
```

Comment:
Should we have the slow mode banner?  We should be consistent about it one way or another.